docs/visualization/.pages

```diff
@@ -17,7 +16,6 @@ nav:
             - Latency histogram: visualization/visualization_api/latency_histogram
 
     - Helper APIs:
-        - List:
-            - Usage of LTTngEventFilter: visualization/helper_api/lttng_event_filter
-            - Path Verification: visualization/helper_api/path_verify
-            - Detailed Behavior Investigation Methods: visualization/helper_api/scope_switch_method
+        - Usage of LTTngEventFilter: visualization/helper_api/lttng_event_filter
+        - Path Verification: visualization/helper_api/path_verify
+        - Regex expressions of get_callbacks(): visualization/helper_api/regex_expressions_get_callbacks
```

Comment:
@takeshi-iwanari 
I adapted your comment.
https://github.com/tier4/CARET_doc/pull/95#discussion_r999227352
> Do we really need "List" here?
> Code like the following doesn't work? Or, does it become strange appearance?

I had thought it couldn't be done well.
But it was OK.